src/localization/generalStrings.json

```diff
@@ -5,7 +5,7 @@
     "idle_log_out_interval": "Idle logout interval (minutes):",
     "sync_interval_minutes": "Sync interval (minutes):",
     "problem_connecting_please_try_again": "There was a problem connecting with the server for this patient. Please ensure you have an internet connection and try again.",
-    "enter_patient_details": "Enter the patient's details to start searching for them",
+    "enter_patient_details": "Enter the patient's details, or scan a QR code to start searching for them",
```

Comment:
Would the QR code scan initiate a 'search'? Surely there's a one to one relationship with the codes?
We also haven't addressed "Search online" and "New Patient" ( or addressed the Inconsistent Capitalisation! )

I think either revert back to "Enter the patient's details to start searching for them"
or
"Enter the patient's details to start searching for them, or for other options, use the buttons above "

Reply:
I didn't even notice the inconsistent Capitalisation until now, and now I can't un-see it 😬 .

Hmm, that's a good point about the wording though. Will update. I guess the QR option is more of a lookup rather than a search, a line to defer to all 3 buttons as 'other options' sounds good to me.

Reply:
Capitalisation: 9e97a54b 
Wording: 45f38049